lib/helpers/asset-helper.js

```diff
@@ -9,7 +10,14 @@ module.exports = {
   },
 
   read: function (assetPath) {
-    return fs.readFileSync(path.resolve(__dirname, '..', 'assets', assetPath)).toString();
+    var parts = assetPath.split('/');
+    var temp = helpers.path.getPath(parts[0].replace(/s$/, '')) + '/' + parts[1];
```

Comment:
nvm. we are stripping the trailing s because the helper adds an additional one. Can you move the `parts[0].replace(/s$/, '')` part to an extra variable called type or something?